### ISSUE TITLE
Crashes Extension Host

### ISSUE THREAD

**Issue by ooliver1** [zabel-xyz/local-history#152]
Type: <b>Bug</b>

Ran an extension bisect and found this extension is the reason the extension host keeps restarting.

Extension version: 1.8.1
VS Code version: Code 1.76.1 (5e805b79fcb6ba4c2d23712967df89a089da575b, 2023-03-08T16:31:31.010Z)
OS version: Linux x64 5.19.0-35-generic
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-11400 @ 2.60GHz (12 x 4199)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|3, 3, 2|
|Memory (System)|15.44GB (9.90GB free)|
|Process Argv|--unity-launch --crash-reporter-id a04fd588-d1ad-4587-8db1-e67985c7bc52|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|wayland|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
pynewvextcfv2:30681851
azure-dev_surveyonecf:30548226
pyindex848:30662994
nodejswelcome1:30587005
2e4cg342:30602488
pyind779:30671433
f6dab269:30613381
vscrp:30673768
pythonsymbol12:30671437
a9j8j154:30646983
vsctsb:30677850
pythonb192:30669360
funwalk2:30682974
pythonms35:30680984

```

</details>

**Comment by fabienmoritz** [comment]
This library is unmaintained :(

**Comment by ooliver1** [comment]
This is not a library.

Edit: sorry for saying it isn't a library instead of agreeing with an extension being a library?

**Comment by fabienmoritz** [comment]
> This is not a library.

OK boomer

**Comment by ooliver1** [comment]
> > This is not a library.
> 
> OK boomer

Try checking my README/profile first.

**Comment by fabienmoritz** [comment]
Right, you're a technically bright but socially inept kid, I stand corrected. The important word of my answer was "unmaintained".
